auditlog/models/rule.py

```diff
@@ -502,6 +542,19 @@ def create_logs(
         model_id = self.pool._auditlog_model_cache[res_model]
         auditlog_rule = self.env["auditlog.rule"].search([("model_id", "=", model_id)])
         fields_to_exclude = auditlog_rule.fields_to_exclude_ids.mapped("name")
+
+        vals = {
+            "model_id": model_id,
+            "method": method,
+            "user_id": uid,
+            "http_request_id": http_request_model.current_http_request(),
+            "http_session_id": http_session_model.current_http_session(),
+        }
```

Comment:
Is there a reason you did not eliminate the redefinition of `vals` in line 561 as in https://github.com/OCA/server-tools/pull/2768/files#diff-312eab5bf54da620ee733a3b9c034685fef15546d8602d851a92eda2227a80b1L527-R582?